modules/builder-prod/install

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+go version
+export GOFLAGS=-mod=vendor
+
+OPERATOR_PATH=/usr/local/bin/kie-cloud-operator
+UI_PATH=/usr/local/bin/console-cr-form
+
+cd $REMOTE_SOURCE_DIR/app && pwd
+
+go generate ./...
+go vet ./...
+go test ./...
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -mod=vendor -a -o ${OPERATOR_PATH} ./cmd/manager
```

Comment:
@rgdoliveira do you remember if the operator is build for power pc as well?

Reply:
@spolti the RHPAM operator is built only for x86_64 - so the PR is correct. The operator that is built also for ppc64le is the Kogito operator.